### PR TITLE
Ignore `p` and `p-span` components in add at macro script

### LINKED ISSUE #267: "add at macro script" should ignore `p-span` & `p` components
The rc/p & deprecated rc/p-span components do not support the keyword arguments, only child components and an optional map as the first argument for styling etc. The add `at` macro script currently injects the :src keyword as first and second args breaking optional first arg map e.g.

```clojure
[rc/p
   {:class "instructions"}
  "Some instructions for the user"]
```
becomes
```clojure
[rc/p 
   :src (at)
   {:class "instructions"}
  "Some instructions for the user"]
```

### PR DESCRIPTION
**Pull request by MawiraIke**
Fixes #267

### CODE REVIEW

File: scripts/add-at-macro/src/add_at_macro/core.clj [hunk 1]
```diff
@@ -242,12 +242,25 @@
           z/up))
     loc))
 
+(defn re-com-component?
+  "Given a string and an alias, finds if the string is referring to a re-com component that should not be
+   added the `:src` annotations. i.e `p` or `p-span`.
+
+   For example
+   - Given the string `rc/p` and the alias `rc`, will return true
```
Comment (superstructor) [inline scripts/add-at-macro/src/add_at_macro/core.clj:250]: won't it be `false` ?

File: scripts/add-at-macro/src/add_at_macro/core.clj [hunk 2]
```diff
@@ -242,12 +242,25 @@
...
+   - Given the string `rc/p-span` and the alias `rc`, will return true
```
Comment (superstructor) [inline scripts/add-at-macro/src/add_at_macro/core.clj:251]: won't it be `false` ?

File: scripts/add-at-macro/src/add_at_macro/core.clj [hunk 3]
```diff
@@ -242,12 +242,25 @@
...
+   - Given any other string such as `rc/box` and an alias such as `rc` will return false"
```
Comment (superstructor) [inline scripts/add-at-macro/src/add_at_macro/core.clj:252]: won't it be `true` ?

File: scripts/add-at-macro/src/add_at_macro/core.clj [hunk 4]
```diff
@@ -242,12 +242,25 @@
           z/up))
     loc))
 
+(defn re-com-component?
```
Comment (superstructor) [inline scripts/add-at-macro/src/add_at_macro/core.clj:245]: fn needs a better name. 

maybe re-com-kwargs-component? or component-accepts-kwargs?

**Review by superstructor — CHANGES_REQUESTED** [review]

**Comment by MawiraIke** [comment]
@superstructor thanks, I have resolved the issues now. Should be good to go.

**Comment by superstructor** [comment]
Thanks @MawiraIke 👍🏻 😄

**Pull request merged**